recipe/meta.yaml

```diff
@@ -23,12 +23,11 @@ requirements:
     - setuptools
     - versioningit
   run:
-    - ambertools >=22
```

Comment:
Awesome - out of curiosity `openfe` probably still has this as hard dependency? AFAICT it's not listed there but would at least be included in this path (if not more?)

`openfe` <- `openfe-base` <- `openmmforcefields` <- `openff-toolkit` <- `ambertools`